WSDOTAgent/GirderScheduleChapterBuilder.cpp

```diff
@@ -683,6 +684,34 @@ rptChapter* CGirderScheduleChapterBuilder::Build(const std::shared_ptr<const WBF
       }
    }
 
+   Float64 tft = pIGirder->GetTopFlangeThickening(segmentKey);
+   if (!IsZero(tft))
+   {
+       GET_IFACE2(pBroker, IPointOfInterest, pPoi);
+       INIT_UV_PROTOTYPE(rptPointOfInterest, location, pDisplayUnits->GetSpanLengthUnit(), true);
+       INIT_UV_PROTOTYPE(rptLengthUnitValue, thickness, pDisplayUnits->GetComponentDimUnit(), true);
+
+       PoiList tfPoi;
+       pPoi->GetPointsOfInterest(segmentKey, POI_TENTH_POINTS | POI_SPAN, &tfPoi);
+
+       FlangeIndexType nFlanges = pIGirder->GetTopFlangeCount(girderKey);
+
+       (*pTable)(++row, 0) << _T("Top Flange Thickness:");
+       pTable->SetColumnSpan(row, 0, 2);
+
+       (*pTable)(++row, 0) << _T("Location:");
+       (*pTable)(row, 1) << _T("Thickness:");
+
+       for (const auto& poi : tfPoi)
+       {
+           for (FlangeIndexType i = 0; i < nFlanges; i++)
```

Comment:
This loops over all top flanges, but there is only on column for top flange thickness. The table isn't going to look right for beams with multiple top flanges (U-Beams).

The internal data model supports only one top flange thickening per girder so you can assume the thickening is the same for all top flanges.

No need to loop over nFlanges. 

Assert that the beam has at least 1 top flange.